src/navbar/navbar.tsx

```diff
@@ -52,9 +59,21 @@ const Component = <T,>(
     const [showDrawer, setShowDrawer] = useState<boolean>(false);
     const [showOverlay, setShowOverlay] = useState<boolean>(false);
     const isStretch = layout === "stretch";
+    const elementRef = useRef<HTMLDivElement>();
 
     const { primary = DEFAULT_RESOURCES.primary, secondary } = resources;
 
+    useImperativeHandle(
+        ref,
+        () =>
```

Comment:
```suggestion
        () => ({
            ...elementRef.current,
            dismissDrawer: () => {
                dismissDrawer();
            }
        })
```

Reply:
actually using `Object.assign` is required, the spread syntax loses the div element attributes. this issue was encountered with `Accordion.Item`